app/models/image.rb

```diff
@@ -30,6 +30,10 @@ def image_linktag
     ApplicationController.helpers.image_linktag(file, title)
   end
 
+  def image_on_disk
+    ActiveStorage::Blob.service.path_for(file.key)
```

Comment:
Proposal to avoid full path exposure:

  def image_on_disk
    full_path = ActiveStorage::Blob.service.path_for(file.key)
    full_path.gsub(Rails.root.to_s, '')
  end

and in the packing process then Rails.root + <PATH> --> Rails.root + image[:image_on_disk]

Reply:
Fixed